modules/utils/src/chains.json

```diff
@@ -404,7 +408,7 @@
         "symbol": "BNB",
         "mainnetEquivalent": "0xB8c77482e45F1F44dE1745F52C74426C631bDD52"
       },
-      "0xc825e75837a3f10e6cc7bda1b85eaac572ac3b8d": {
+      "0x532197ec38756b9956190b845d99b4b0a88e4ca9": {
```

Comment:
Is this your PAID token address on BSC testnet

Reply:
> Is this your PAID token address on BSC testnet

Thanks for your review.

Yeah, they both have the same address:
https://testnet.bscscan.com/token/0x532197ec38756b9956190b845d99b4b0a88e4ca9
https://bscscan.com/address/0x532197ec38756b9956190b845d99b4b0a88e4ca9

This is because we are using smart contract upgradable proxy